geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreQueryTest.scala

```diff
@@ -376,25 +371,6 @@ class AccumuloDataStoreQueryTest extends Specification with TestWithMultipleSfts
       (System.currentTimeMillis() - start) must beLessThan(30000L)
     }
 
-    "avoid deduplication when possible" in {
```

Comment:
Is deduplication all done?

Reply:
yeah, it was for the non-point z3/z2 indices